en/micro-integrator/docs/use-cases/examples/protocol-switching/switching_from_UDP_to_HTTPS.md

```diff
@@ -17,8 +18,57 @@ Following are the integration artifacts (proxy service) that we can used to impl
             <property name="OUT_ONLY" value="true"/>
         </inSequence>
     </target>
+    <publishWSDL key="conf:UDP_WSDL/sample_proxy_1.wsdl" preservePolicy="true"/>
     <parameter name="transport.udp.port">9999</parameter>
     <parameter name="transport.udp.contentType">text/xml</parameter>
-    <publishWSDL uri="file:repository/conf/sample/resources/proxy/sample_proxy_1.wsdl"/>
 </proxy>
 ```
+
+## Build and Run
+
+Create the artifacts:
+
+1. [Set up WSO2 Integration Studio](../../../../develop/installing-WSO2-Integration-Studio).
+2. [Create an ESB Solution project](../../../../develop/creating-projects/#esb-config-project).
+3. Add [sample_proxy_1.wsdl](https://github.com/wso2-docs/WSO2_EI/blob/master/samples-protocol-switching/sample_proxy_1.wsdl) as a [registry resource](../../../../develop/creating-artifacts/creating-registry-resources). 
```

Comment:
We may have to mention the registry location

Reply:
+1 done